.github/workflows/build_library.yml

```diff
@@ -10,31 +10,54 @@ env:
 
 jobs:
   linux:
+    timeout-minutes: 30
     strategy:
       fail-fast: false
       matrix:
         build-arch:
-          - manylinux2010_i686
-          - manylinux2010_x86_64
-          - manylinux2014_aarch64
+          - { image: 'manylinux2014_x86_64', target: 'i686' }
+          - { image: 'manylinux2014_x86_64', target: 'x86_64' }
+          - { image: 'manylinux2014_x86_64', target: 'aarch64' }
```

Comment:
See comment below in the builder script. We do not actually use the image declared here, and I would suggest we remove `image` from here.